includes/settings/settings.php

```diff
@@ -191,26 +119,11 @@ function() {
 	 * Fired by `admin_menu` action.
 	 *
 	 * @since 2.0.3
-	 * @access public
+	 * @access private
 	 */
-	public function register_knowledge_base_menu() {
-		add_submenu_page(
-			self::PAGE_ID,
-			'',
-			__( 'Getting Started', 'elementor' ),
-			'manage_options',
-			'elementor-getting-started',
-			[ $this, 'elementor_getting_started' ]
-		);
-
-		add_submenu_page(
-			self::PAGE_ID,
-			'',
-			esc_html__( 'Get Help', 'elementor' ),
-			'manage_options',
-			'go_knowledge_base_site',
-			[ $this, 'handle_external_redirects' ]
-		);
+	private function register_knowledge_base_menu( Admin_Menu_Manager $admin_menu ) {
+		$admin_menu->register( 'elementor-getting-started', new Getting_Started_Menu_Item() );
+		$admin_menu->register( 'go_knowledge_base_site', new Get_Help_Menu_Item() );
```

Comment:
These items should be after the System Info item

Reply:
Done